test/e2e/proxy.go

```diff
@@ -218,15 +226,24 @@ func proxyContext(version string) {
 			defer errLock.Unlock()
 			errs = append(errs, s)
 		}
-		for i := 0; i < proxyAttempts; i++ {
-			for path, val := range expectations {
+		d := time.Since(start)
+		framework.Logf("setup took %v, starting test cases", d)
+		numberTestCases := len(expectations)
+		totalAttempts := numberTestCases * proxyAttempts
+		framework.Logf("%v cases, %v attempts per case, %v total attempts", numberTestCases, proxyAttempts, totalAttempts)
+
+		for path, val := range expectations {
```

Comment:
I didn't write it. I just reversed the order of the loops so the output was more coherent. This runs the all the attempts against one proxyPath before moving on. See up above in the old version of lines 221&222.